app/controllers/application_controller.rb

```diff
@@ -25,10 +25,8 @@ def current_user
   def after_sign_in_path
     if current_user.support_user?
       support_root_path
-    elsif current_user.memberships.many?
-      organisation_index_path
     else
-      root_path
+      organisation_index_path
```

Comment:
I just saw this in the helper now too. Since you're touching it, can we rename this to `organisations_path` to match Rails' conventions please?

Reply:
We can but I thought `organisations_path` is like a reserved name but I guess we have placements and claims org path so for now we should be fine